step3/chat/ChatController.js

```diff
@@ -0,0 +1,36 @@
+const storage = new(require('../dataStores/Storage'));
+
+class ChatController {
+
```

Comment:
No i teraz pytanie - dlaczego ten obiekt nie może być routerem? jest z nim bardzo silnie powiązany poprzez parametry req i res, i nie dodaje przez to jakiejś dodatkowej abstrakcji. Zastanawiam się czy rozpakowanie parametrów w routerze i tutaj wykorzystywanie tylko takich pojęć dziedzinowych nie byłoby lepsze.